scipy/signal/signaltools.py

```diff
@@ -1378,6 +1379,11 @@ def medfilt(volume, kernel_size=None):
         An array the same size as input containing the median filtered
         result.
 
+    Raises
+    -------
+        UserWarning
```

Comment:
I just grepped through our code base and couldn't find any example of a `Warning` being mentioned in a `Raises` section of a docstring. The identation is a little strange here too, though it doesn't seem to affect the render.

Reply:
I don't know if it might be better to place in a `Notes` section or something like that--let's wait for another person to chime in on that.

Reply:
Yeah, this would go under a `Warns` section rather than `Raises` one.

As you noted, we do not seem to have been documenting such warnings in SciPy, so not sure if we want to do it here or not. NumPy itself does seem to use it occasionally, but not consistently.

Reply:
@grlee77, it will go to `Warns` section then